packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuInput.tsx

```diff
@@ -53,8 +53,7 @@ const StyledRightContainer = styled.div`
 
 const StyledErrorDiv = styled.div`
   color: ${({ theme }) => theme.color.red};
-  padding: 0 ${({ theme }) => theme.spacing(2)}
-    ${({ theme }) => theme.spacing(1)};
+  padding: 0 ${({ theme }) => theme.spacing(2)};
```

Comment:
Since the original issue doesn't mention padding, could you please add a comment to explain the issue you've found/are fixing here? Thanks!

Reply:
Why did you resolve @Hitarthsheth07? Thanks